tests/s3/test_multipart_getput.py

```diff
@@ -758,3 +757,102 @@ def test_multipart_upload_test_28528(self):
         self.get_obj_compare_checksums(self.bucket_name, self.object_name, resp[1]["ETag"])
         self.log.info("Stop and validate parallel S3 IOs")
         self.log.info("ENDED: Test List multipart with 10000 parts")
+
+    @pytest.mark.tags('TEST-40255')
+    @pytest.mark.s3_ops
+    @CTFailOn(error_handler)
+    def test_multipart_upload_test_40255(self):
+        """
+        This test is for overwriting multipart uploaded object by simple object
+        """
+        mp_config = MPART_CFG["test_28539"]
```

Comment:
@archanabpatil  Are you expecting the config to be updated before the test run?

Reply:
Shared configs should be taken from either Params or fixtures depending on their spread.

Reply:
> Are you expecting the config to be updated before the test run?
No , I want to use same config as test_28539

Reply:
Its just at 2 tests in this file and the config is same for these 2 tests. So I just reused the same config